flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -420,31 +406,6 @@ static int32_t vtol_fsm_static()
  * @{
  */
 
-//! The currently configured navigation mode. Used to sanity check configuration.
-static enum vtol_nav_mode vtol_nav_mode;
-
-/**
- * General methods which based on the selected @ref vtol_nav_mode calls the appropriate
- * specific method
- */
-static int32_t do_default()
```

Comment:
The do_default was scary and there were a few cases where it could do the right thing.  I made sure each state machine had a static_fn defined for all states.  It doesn't use more memory and it's good to be explicit.